### PR TITLE
Cfquantity

### PR DESCRIPTION
**Pull request by dinossimpson**

### CODE REVIEW

File: test_juliantxt.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 sp = pyspeckit.Spectrum('julian_infrared.txt')
 
 # set up units properly
-sp.xarr.units='angstroms'
+sp.xarr._unit='angstroms'
```
Comment (keflavich) [inline test_juliantxt.py]: this can be `angstrom` now

**Comment by dinossimpson** [comment]
@keflavich I have commented out in alberto_example.py and vega_echelle_example.py some lines that are saving the figure (plotter.figure.savefig()) because it was causing the LaTeX error locally. I have a feeling these might pass in travis.

**Comment by keflavich** [comment]
OK.  We will want to un-comment those lines before we merge into master, but it is OK for them to live in commented out versions on these branches

**Comment by keflavich** [comment]
Hey @dinossimpson  - I merged the cfquantity branch on pyspeckit but I guess I never merged this.  Could you test locally and let me know if it's ready to merge?

**Comment by dinossimpson** [comment]
These are the tests I currently use locally for the cfquantity_patch branch and all other commits on top of them are in different branches, namely in test_ammonia and test_moments. I'm noting to remove any units assignments to reduce the warnings and then they should be ok.

**Comment by keflavich** [comment]
I think there was some sort of red herring... I checked the tests locally and nothing was wrong, so I don't know why merging this made files disappear.

**Comment by keflavich** [comment]
Nope.  After merging, I get:
```
*****test_nh3_loading_regression.py*****
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "pyspeckit/__init__.py", line 33, in test

  File "pyspeckit/tests/run_tests.py", line 102, in test_everything
    execfile(os.path.join(dir_prefix,'test_nh3_loading_regression.py'))
IOError: [Errno 2] No such file or directory: '/private/var/folders/_r/c4nkvmpn4vx6qc91qygz8h740000gr/T/pyspeckit-test-pQhBrF/lib.macosx-10.5-x86_64-2.7/pyspeckit/tests/../../examples/test_nh3_loading_regression.py'
```

Before merging, there was no problem.  This is not possible.

**Pull request merged**